distance_to_car_front: 4.0  # m
braking_reaction_time: 1.6
waypoint_interval: 1.0          # m
default_acceleration: 1.0       # m/s2
default_deceleration: 1.0       # m/s2
max_deceleration: 10.0          # m/s2
stopped_speed_limit: 1.0        # m/s

safety_box_width: 2.7           # m
safety_box_length: 5.0          # m
default_left_width: 1.4         # m
default_right_width: 1.4        # m
local_path_length: 100

lanelet2_map_visualizer:
  use_map_extraction: False # extract smaller map area around the ego vehicle for visualization
  map_extraction_distance: 500 # m - distance from ego vehicle to extract map data

waypoint_loader:
  output_frame: map
  wp_left_width: 1.4
  wp_right_width: 1.4

lanelet2_global_planner:
  output_frame: map
  distance_to_goal_limit: 4.0

local_path_extractor:
  publish_rate: 10                        # Hz
  lookahead_distance: 10                  # m
  distance_to_lookahead_path_limit: 2.5   # m
collision_points_manager:
  braking_safety_distance_obstacle: 4     # m
  braking_safety_distance_goal: 0.1       # m
  braking_safety_distance_stopline: 2     # m
simple_speed_planner:
  synchronization_queue_size: 4
  synchronization_slop: 0.15